### DISCUSSION TITLE
Help on integrating SonarCloud with Defect

### DISCUSSION THREAD

**Discussion by MathzRocha** [DefectDojo/django-DefectDojo#10814] [Q&A] [answered]
Hi guys, I'd like some help, I need to integrate my Cloud sonar with Defect. I just need Dojo to pull the vulnerabilities found by Sonar Cloud into Defect, could someone help me with this integration?

**Comment by kiblik** [comment, marked as answer]
Hi @MathzRocha. Steps are described in:
- https://documentation.defectdojo.com/integrations/parsers/api/
- https://documentation.defectdojo.com/integrations/parsers/api/sonarqube/

Are you struggling with some specific steps?
